OneTimeKit/Models/OTPBase.m

```diff
@@ -186,6 +187,14 @@ - (NSString *)password {
     return [self passwordForFactor:[self factor]];
 }
 
+- (NSString *)secret {
+    return [self.key base32EncodedStringWithOptions:NSDataBase32EncodingOptionsNoPad];
+}
+
+- (NSString *)qrString:(NSArray *)input {
+    return [NSString stringWithFormat: @"otpauth://totp/%@:%@?secret=%@&issuer=%@", input[0],input[1],self.secret,input[0]];
```

Comment:
This is currently implemented in https://github.com/leptos-null/OneTime/blob/88395900c67852bb9e7597c2bdae5a2a150b1844/OneTimeKit/Models/OTBag.m#L166